Assets/MRTK/StandardAssets/Shaders/MRTK.Shaders.sentinel

```diff
@@ -0,0 +1 @@
+ver: 0
```

Comment:
Since we're unable to use ProjectPreferences due to the dependency direction, I wonder if we could embed some sort of "ignore" flag here, so we can still allow developers to stop the pop-up if they've customized

Reply:
hmm... let me see what I can come up with...

Reply:
implemented an ignore file